src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -247,9 +211,54 @@
                         {:opacity bg-opacity}
                         styles/backdrop)}]]
+             ;; VVVVVVV
+             ;; This is repeated because gesture-handler doesn't play nicely when it's conditionally
+             ;; rendered with reanimated
              (if show-handle?
-               [bottom-sheet-comp
-                [gesture/gesture-detector {:gesture bottom-sheet-gesture}
-                 [handle-comp window-width]]]
+               [reanimated/view
+                {:style (reanimated/apply-animations-to-style
+                         {:transform [{:translateY translate-y}]}
+                         {:width  window-width
+                          :height window-height})}
+                [rn/view {:style styles/container}
+                 (when selected-item
+                   [rn/view {:style (styles/selected-background)}
+                    [selected-item]])
+                 [rn/view {:style (styles/background)}
+                  [rn/keyboard-avoiding-view
+                   {:behaviour (if platform/ios? :padding :height)
+                    :style     {:flex 1}}
+                   [rn/view
+                    {:style     (styles/content-style insets)
+                     :on-layout (when-not (and
+                                           (some? @content-height)
+                                           (> @content-height 0))
+                                  on-content-layout)}
+                    children]]
+                  [gesture/gesture-detector {:gesture bottom-sheet-gesture}
+                   [handle-comp window-width]]]]]
                [gesture/gesture-detector {:gesture bottom-sheet-gesture}
-                [bottom-sheet-comp [handle-comp window-width]]])]))])]))
+                [reanimated/view
+                 {:style (reanimated/apply-animations-to-style
+                          {:transform [{:translateY translate-y}]}
+                          {:width  window-width
+                           :height window-height})}
+                 [rn/view {:style styles/container}
+                  (when selected-item
+                    [rn/view {:style (styles/selected-background)}
+                     [selected-item]])
+                  [rn/view {:style (styles/background)}
+                   [rn/keyboard-avoiding-view
+                    {:behaviour (if platform/ios? :padding :height)
+                     :style     {:flex 1}}
+                    [rn/view
+                     {:style     (styles/content-style insets)
+                      :on-layout (when-not (and
+                                            (some? @content-height)
+                                            (> @content-height 0))
+                                   on-content-layout)}
+                     children]]
+
+                   [handle-comp window-width]]]]]
+               #_[gesture/gesture-detector {:gesture bottom-sheet-gesture}
```

Comment:
Why keep the zombie code here? Note that zombie code is worse than dead code, it gently attacks developers every now and then, confuses them, and drags them into unnecessary discussions ;)

Seriously, if there's a need to keep the zombie code, please, always add a comment explaining why. But preferably, just remove them and we can rely on git for going back in history.

Reply:
No worries :mechanical_arm: